fix: add requirement on Nonce payload to contain an actual random value (#23)
peppelinux/draft-demarco-oauth-nonce-endpoint: draft-demarco-oauth-nonce-endpoint.md

}
~~~~

> Please note that the values represented in the previous examples MAY depend on domain specific requirements and MUST NOT be intended as normative.

Reviewer: ```suggestion
Please note that the values represented in the previous examples are informative.
```
